Flatten/Flatten.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
Create a folder named "Save" and bring the sample inside the folder

**Save** 
* Flatten

Reply:
addressed